src/silo/common/bidirectional_string_map.h

```diff
@@ -0,0 +1,91 @@
+#pragma once
+
+#include <cstdint>
+#include <functional>
+#include <optional>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#include <yaml-cpp/yaml.h>
+#include <boost/serialization/split_member.hpp>
+
+#include "silo/common/types.h"
+
+namespace silo::common {
+
+class BidirectionalStringMap {
+   std::vector<std::string> id_to_value;
```

Comment:
so just to clarify `id_to_value` is a vector of values ordered according to Idx, so I assume that Idx starts at 0 and goes to N(=#values)? Maybe it would be good to comment this